client/src/main/java/org/apache/nemo/client/ClientEndpoint.java

```diff
@@ -109,20 +114,7 @@ private boolean waitUntilConnected(final long timeout,
    * @return {@code true} if the manager set.
    */
   private boolean waitUntilConnected() {
-    connectionLock.lock();
-    try {
-      if (driverEndpoint.get() == null) {
-        // If the driver endpoint is not connected, wait.
-        driverConnected.await();
-      }
-      return true;
-    } catch (final InterruptedException e) {
-      e.printStackTrace(System.err);
-      Thread.currentThread().interrupt();
-      return false;
-    } finally {
-      connectionLock.unlock();
-    }
+    return waitUntilConnected(0, TimeUnit.MILLISECONDS);
```

Comment:
Do we still need this function now?

Reply:
It's not used at the moment, but I think it's correct to guide the users to some point where there is no timeout with the method.